lib/datahub-client/data_platform_catalogue/client/search.py

```diff
@@ -107,35 +113,28 @@ def search(
     def _parse_search_results(self, response) -> Tuple[list, list]:
         page_results = []
         malformed_result_urns = []
+
+        EntityMapping = namedtuple('EntityMapping', ['result_type', 'parsing_function'])
```

Comment:
I think you can move `EntityMapping` to module level to avoid redefining it each time